hawkbit-repository/hawkbit-repository-core/src/main/java/org/eclipse/hawkbit/event/EventType.java

```diff
@@ -166,6 +176,15 @@ public class EventType {
         TYPES.put(44, TargetTypeCreatedEvent.class);
         TYPES.put(45, TargetTypeUpdatedEvent.class);
         TYPES.put(46, TargetTypeDeletedEvent.class);
+
+        // processing events
+        TYPES.put(47, ProcessingTargetCreatedEvent.class);
```

Comment:
maybe leave some space for next db / fanout events? e.g. start heir order from 1000?